onboardingtaskapp.client/src/components/Customer/Edit.jsx

```diff
@@ -1,23 +1,39 @@
 import { React, useState, useEffect } from 'react';
-import { Modal, Button } from 'semantic-ui-react';
+import { Modal, Button, Message } from 'semantic-ui-react';
 
-export default function Edit({ item }) {
+export default function Edit({ item, isUpdated }) {
 
+    // states
     const [open, setOpen] = useState(false);
-    const handleOpen = () => setOpen(true);
-    const handleClose = () => setOpen(false);
-
+    const [message, setMessage] = useState(null); 
 
     const [formData, setFormData] = useState({        
         name: '',
         address: '',
     });
-
+      
     useEffect(() => {
-        
-        const fetchCustomer = async () => {
+
+        if (!item.customer.id) {
+            throw new Error('Invalid Id');
+        }
+                
+        fetchCustomer(item);
+
+    }, [item.customer.id]);
+
+    // const
+    const API_END_POINT = `https://onboardsite.azurewebsites.net/api/Customer/`;
+
+    // methods
+    const handleDisplayModal = () => {
+        setOpen(!open);
+        setMessage(null);
+    };
+       
+    const fetchCustomer = async () => {
             try {
-                const response = await fetch(`https://localhost:7207/api/Customer/${item.customer.id}`);
+                const response = await fetch(`${ API_END_POINT + item.customer.id }`);
 
                 console.log(response);
```

Comment:
Can you remove this? We need to remove console.log in product level so users won't be able to see any unwated data. Thanks